### ISSUE TITLE
error: invalid command 'osx'

### ISSUE THREAD

**Issue by svandragt** [pybee/briefcase#8]
I installed briefcase using `pip install briefcase`, but then I couldn't run `briefcase` as I get:

```
(ENV)$ briefcase
-bash: briefcase: command not found
```

Now when I try and build the toga webkit browser test app I get the following error:

```
(ENV) $ python setup.py osx
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: invalid command 'osx'
```

This is probably because the briefcase commands have not been added to my virtual environment? OS X 10.11.4; Python 3.5.1

**Comment by Cerise1** [comment]
Hi Svan 

Did you find a solution to this problem cause i have the exact same problem with python setup.py ios command .. 
Thanks

**Comment by freakboy3742** [comment]
@Cerise1 - If you've installed recently, you might have been caught by a recent update  - I've renamed "osx" to "macos" to reflect Apple's own renaming of the platform. 

You may also have problems if you've installed via `python setup.py install`,  rather than `pip install`.

Let me know if either of these correct the problem for you. Leaving this open as a marker to update the docs to say "macos", not "osx"

**Comment by Cerise1** [comment]
hi freakboy, thanks for your quick reply !

My problem was not with the osx command but with the iOS command cause i try to create an application for ios !

And yes i ve installed briefcase with pip install briefcase ! 

when i try the command i got : 

` python setup.py ios
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: invalid command 'ios'`

and when i try `python setup.py --help-commands` it shows only some native command like build sdist etc ...

**Comment by freakboy3742** [comment]
Ah - another option: What version of `setuptools` do you have installed? Older versions are known to have some problems installing distutils hooks. Try the following:

    pip uninstall briefcase
    pip install --upgrade setuptools
    pip install briefcase

**Comment by Cerise1** [comment]
hum pip gave me an error so i upgrade with easy-install and it worked !

But then exact same error with ios macos or android !

to make sure i am not wrong on the procedure : 
i have a directory on my linux distro with a main.py file wich lauch au python script, i also created a setup.py file as described in the briefcase readme on the same directory. 
Then i go on the terminal cd to the directory and run `python setup.py ios` 
then i get the error :(

**Comment by marcmolla** [comment]
I had the same error because I was using distutils.core instead of setuptools setup, which is required for handling the iOS, macos and android entry points.

**Comment by freakboy3742** [comment]
I'm guessing this means we should add `setuptools>=27.0` as an `install_requires` entry. We should probably update the pip requirement to `pip>=8.1` while we're at it.

**Comment by freakboy3742** [comment]
Fixed by #18
